services/harmony/app/frontends/workflow-ui.ts

```diff
@@ -198,6 +198,12 @@ function jobRenderingFunctions(logger: Logger, requestQuery: Record<string, any>
         return this.request;
       }
     },
+    jobLabelsDisplay(): string {
+      return this.labels.map((label) => {
```

Comment:
I wonder if we should sort the labels here?

Reply:
Regarding label sorting -- this is where we assign the labels property in `queryAll()` :
```
 const jobs: Job[] = items.data.map((j: JobWithLabels) => {
      const job = new Job(j);
      if (includeLabels && j.label_values) {
        job.labels = j.label_values.split(',');
      } else {
        job.labels = [];
      }
      return job;
    });
```
I'm thinking it's safe to assume that if you've set i`ncludeLabels=true` in `queryAll` that you're going to display the labels. So rather than add a new parameter (`sortLabels`), I think we should just sort alphabetically here by default. However, the runtime might not be great (sorting an array for each job returned), but I'm not sure there's a better alternative unless we restructure the join query somehow to include an alphabetical sort.

Reply:
`string_agg` has an order by option, so perhaps that'll do the trick.

Reply:
That worked.